source/stylesheets/corporate/sections/_team.sass

```diff
@@ -4,8 +4,12 @@
 
 #team
   padding-bottom: $sectionsMargin
-  .col:hover .avatar
-    @include filter(none)
+  .col
+    &:hover .avatar
+      @include filter(none)
+    &:nth-child(4)
+      margin-left: 0
+      margin-right: 0
```

Comment:
Care, this is not bootstrap, it's flexbox, that's why your margin: 0 (left, right) is working that way. And the last team member is strangely placed there, no?